highfidelity/hifi
# Case 20200: Update EntityScriptingInterface to not find local entities (#2)

Same as https://github.com/highfidelity/hifi/pull/14547 but it doesn't break everything

Test plan:
- run all the engine/interaction tests from hifi_tests
- Run [this](https://gist.githubusercontent.com/SamGondelman/33da7f2cb8d798f934aa6c11ee68bd64/raw/68cdfb200c626495cfdb17c22d2b6f275be67cdf/FindEntitiesTest.js).  You'll see this:
![image](https://user-images.githubusercontent.com/7650116/49762359-611a6a00-fc7e-11e8-85de-26f3c700e2b6.png)
- Verify that the hand laser works on the tablet and that you can click on things.
